src/page/sign-in-step.page.ts

```diff
@@ -16,11 +16,11 @@ export class SignInPage {
   }
 
   public async logIn(userName: string, passwd: string): Promise<void> {
-    await browser.wait(ExpectedConditions.presenceOf(this.userEle), 10000, 'It is not possible to access.');
+    await browser.wait(ExpectedConditions.presenceOf(this.userEle), 50000, 'It is not possible to access.');
```

Comment:
Why the change?

Reply:
It was an unnecessary change, I was testing something and I forgot to remove it.